recipes/tensorflow-text/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "2.17.0" %}
+{% set tensorflow_version = ".".join(version.split(".")[:2]) %}
+
+package:
+  name: tensorflow-text
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/text/archive/v{{ version }}.tar.gz
+  sha256: 55bff24f120024664124c8fdc0ab842ab314bd5ff2791770a80f32bb67cb26d1
+
+build:
+  number: 0
+  skip: true  # [win]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
+    - bazel >=5.2.0
```

Comment:
I would pin to bazel 6 for now, like we do in tensorflow.

Reply:
Greatly appreciate your feedback!